blueocean-core-js/src/js/config.js

```diff
@@ -85,6 +87,13 @@ export default {
         return (typeof config.jenkinsRootURL === 'string' ? config.jenkinsRootURL : '/jenkins');
     },
 
+    getResourceURL() {
+        if (!config.isLoaded) {
+            this.loadUrls();
+        }
+        return (typeof config.resourceUrl === 'string' ? config.resourceUrl : '/jenkins/static');
```

Comment:
for some reason, typeof + string makes me remember some really exotic bug we had... and undefined... wish I could remember.